yarn-project/pxe/src/kernel_prover/kernel_prover.test.ts

```diff
@@ -157,14 +159,6 @@ describe('Kernel Prover', () => {
     }
   });
 
-  it('should throw if call stack is too deep', async () => {
```

Comment:
Don't have to check in the prover. It will fail to simulate and to create the proof.